lessons/06-html-layout/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <title>Jaynen's Blog</title>
```

Comment:
The `<title>` tag needs to go inside of a `<head>` tag 😄 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/head